build/configs/imxrt1050-evk/elf/Make.defs

```diff
@@ -0,0 +1,169 @@
+###########################################################################
```

Comment:
What's the purpose of this new configurations set?

Reply:
It enables the elf loader. The final DA app must be based on this config.

Reply:
not xxx-protected?

Reply:
No. That is for user-kernel separation. 
This one is for app separation and loadable apps.

Reply:
ok. then, How about using "loadable app" or something else? not module name.

Reply:
Basically this config enables:
- ELF loader
- App binary separation
- 2 example elf apps
- Dynamic loading of the 2 apps

So, we just named the configuration as elf to indicate that it consists of elf loader and example loadable elf apps. Maybe we could rename it as **elf_app_example** or **elf_app_separation** ?
When we mention elf, it is understood that it is loadable apps right?

Reply:
I modified the configuration to **loadable_elf_apps**